### PR TITLE
Minesweeper: Remove decimal point from the time label

### PR DESCRIPTION
**Pull request by krkk**
> one of my laziest prs, ngl

The timer was updating fast enough to be a little distracting for me.

old | new
---|---
 |

### CODE REVIEW

File: Userland/Games/Minesweeper/Field.cpp [hunk 1]
```diff
@@ -112,12 +112,12 @@ Field::Field(GUI::Label& flag_label, GUI::Label& time_label, GUI::Button& face_b
...
     , m_on_size_changed(move(on_size_changed))
 {
-    m_timer = add<Core::Timer>();
-    m_timer->on_timeout = [this] {
-        ++m_time_elapsed;
-        m_time_label.set_text(String::formatted("{}.{}", m_time_elapsed / 10, m_time_elapsed % 10));
-    };
-    m_timer->set_interval(100);
+    m_timer = Core::Timer::create_repeating(
+        100, [this] {
```
Comment (AtkinsSJ) [inline Userland/Games/Minesweeper/Field.cpp:116]: If it only shows whole seconds, this can update every 1000 ms instead can't it?

Comment (krkk) [reply Userland/Games/Minesweeper/Field.cpp]: oh, sure.. i kind of forgot about it :sweat_smile:

File: Userland/Games/Minesweeper/Field.cpp [hunk 2]
```diff
@@ -341,8 +341,10 @@ void Field::on_square_clicked_impl(Square& square, bool should_flood_fill)
         return;
     if (square.is_considering)
         return;
-    if (!m_timer->is_active())
+    if (!m_timer->is_active()) {
+        m_timer->on_timeout();
```
Comment (AtkinsSJ) [inline Userland/Games/Minesweeper/Field.cpp:345]: `on_timeout()` is never set now, so this should be unnecessary.

Comment (krkk) [reply Userland/Games/Minesweeper/Field.cpp:345]: `on_timeout()` is still set. It’s just being done by the Timer constructor now.

https://github.com/SerenityOS/serenity/blob/977aa8131055ef4185c6f4fbde6b1412f063f542/Userland/Libraries/LibCore/Timer.cpp#L17-L20

Comment (AtkinsSJ) [reply Userland/Games/Minesweeper/Field.cpp:345]: :yakthonk: It's a bit janky that you need to manually call a timer's callback, but fair enough! It's not related to what you did here.

**Pull request merged**